bvm/ballerina-runtime/src/main/java/org/ballerinalang/jvm/types/BFunctionType.java

```diff
@@ -119,4 +121,9 @@ public String toString() {
     public String getAnnotationKey() {
         return this.typeName;
     }
+
+    public List<String> getParamAnnotationKeys() {
+        return Arrays.stream(this.annotations.getKeys()).filter(key -> key.startsWith("$param$")).collect(
```

Comment:
Convert this to a loop. We don't use streams in the runtime since it has a performance impact.